### DISCUSSION TITLE
Missing Icons

### DISCUSSION THREAD

**Discussion by dexter0381** [zixpo/candybar#115] [Q&A]
Hi,

Is there a way I can create  a mask icon that can mask those icons that are missing?
So that all icons apear the same size and cut?

**Comment by sarsamurmu** [comment]
Two things that you can use

Icon mask - https://github.com/teslacoil/Example_NovaTheme/blob/master/README.md#generating-replacement-icons

Shader.xml - https://iislocus.wordpress.com/2013/12/09/%EF%BB%BFhow-to-use-the-shader-xml

**Comment by dexter0381** [comment]
Hi

I am not a developer. Do you have like a sample?

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by dexter0381** [comment]
I finally get how to use Icon Mask.
However, icons in mask are smaller than those with themed Icons that I
manually created. Is there a way, It can be set even size?

**Reply by sarsamurmu** [reply]
I'm not sure how to control that.
